### DISCUSSION TITLE
Accordion not collapsing on title click in vue 3 component

### DISCUSSION THREAD

**Discussion by realrecordzLab** [twbs/bootstrap#36611] [Q&A]
I'm implementing accordion in a vue 3 app. 
I've noticed that I'm only able to open the accordion but not close them when I click on the title. Is there someone else that have faced the same problem? 

I've included all the needed dependencies in the project and I have this code in my `main.js` file.
```js
import { createApp } from 'vue'
import App from './App.vue'
import './registerServiceWorker'
import router from './router'
import store from './store'
import '@popperjs/core'
import 'bootstrap/dist/css/bootstrap.min.css'
import 'bootstrap/dist/js/bootstrap.min.js'
import '@fortawesome/fontawesome-free/css/all.min.css'
import 'firebaseui/dist/firebaseui.css'

createApp(App)
    .use(router)
    .mount('#app')
```    

In my single file component template, I have this code that will loop some data to create a group of accordion items
```html
                <div ref="postContent" class="accordion accordion-flush" id="accordionFlush">
                    <div class="accordion-item" v-for="( section, index) in contentSections" :key="index">
                        <h2 class="accordion-header" id="flush-header">
                        <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" :data-bs-target="`#flush-${index}`">
                            Item #{{ index + 1 }}
                        </button>
                        </h2>
                        <div :id="`flush-${index}`" class="accordion-collapse collapse" data-bs-parent="#accordionFlush">
                            <div class="accordion-body" v-html="section.sectionBody"></div>
                        </div>
                    </div>
                </div>
```

Anyone have suggestion on how I can fix this?

**Comment by kyftball** [comment]
I had the same issue with vue3 and the collapse component.  I spent way too much time trying to diagnose it.  I thought maybe the events were firing twice because you can see the classes changing but at the end the "show" class reappears.  This makes it look like it never collapses.  I _think_ its happening at target !== transitionElement.  This is ALWAYS false because of the __vnode tags on the element.  Never figured out to fix it so I wrote my own.  Wish this had worked.


![image](https://user-images.githubusercontent.com/15369491/213531313-8aa44d03-53fd-4bb7-93f5-555c04cdca7f.png)
